guides/17_Migration_to_3_0.md

```diff
@@ -163,6 +163,117 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## AudioVideo events
+
+We have removed below `AudioVideo` events in v3.
+
+- videoSendHealthDidChange
+- videoSendBandwidthDidChange
+- videoReceiveBandwidthDidChange
+- videoNotReceivingEnoughData
+- estimatedDownlinkBandwidthLessThanRequired
+
+In 3.x you can still replecate these events funtionality through `metricDidReceive`. However `estimatedDownlinkBandwidthLessThanRequired` can not be easily replicated.
+
+
+```js
+  // Before
+const observer = {
+  videoSendHealthDidChange: (bitrateKbps, packetsPerSecond) => {
+    console.log(`Sending video bitrate in kilobits per second: ${videoUpstreamBitrate} and ${videoUpstreamPacketPerSecond}`);
+  },
+  videoSendBandwidthDidChange: (newBandwidthKbps, oldBandwidthKbps) => {
+    console.log(`Sending bandwidth is ${availableSendBandwidth}, nack count per second is ${nackCountPerSecond}, and old bandwidth is ${this.oldSendBandwidth}`);
+  },
+  videoReceiveBandwidthDidChange: (newBandwidthKbps, oldBandwidthKbps) => {
+    console.log(`Receiving bandwidth is ${availableRecvBandwidth}, and old bandwidth is ${this.oldRecvBandwidth}`);
+  },
+  videoNotReceivingEnoughData: (receivingDataMap) => {
+    console.log(
+      `One or more video streams are not receiving expected amounts of data ${JSON.stringify(
+        Array.from(receivingDataMap.values())
+      )}`
+    );
+  }
+}
+// After
+const observer = {
+  oldSendBandwidthKbs: 0,
+  oldRecvBandwidthKbs: 0,
+  metricsDidReceive: (clientMetricReport) => {
+    const metricReport = clientMetricReport.getObservableMetrics();
+
+    const {
+      videoPacketSentPerSecond,
+      videoUpstreamBitrate,
+      nackCountPerSecond,
+    } = metricReport;
+    const availableSendBandwidthKbs = metricReport.availableOutgoingBitrate / 1000;
+    const availableRecvBandwidthKbs = metricReport.availableIncomingBitrate / 1000;
+    // videoSendHealthDidChange
+    console.log(`Sending video bitrate in kilobits per second: ${videoUpstreamBitrate / 1000} and sending packets per second: ${videoPacketSentPerSecond}`);
+
+    // videoSendBandwidthDidChange
+    if(this.oldSendBandwidthKbs != availableSendBandwidthKbs)
```

Comment:
nit: Please follow the pattern like below for all `if`s:
```
if (this.oldRecvBandwidthKbs != availableRecvBandwidthKbs) {
  console.log(`Receiving bandwidth is ${availableRecvBandwidthKbs}, and old bandwidth is ${this.oldRecvBandwidthKbs}`);
  this.oldRecvBandwidthKbs = availableRecvBandwidthKbs;
}
```

Reply:
Updated